tests/conftest.py

```diff
@@ -529,6 +575,62 @@ def cherrytrack_run_info_response(run_id):
     }
 
 
+@pytest.fixture
+def cherrytrack_destination_plate_response(destination_barcode, source_barcode, run_id):
+    return {
```

Comment:
I think these type of "data" responses should be moved out of this file.

Reply:
Ok